cmd/kubeadm/app/util/config/testdata/defaulting/master/defaulted.yaml

```diff
@@ -113,6 +113,7 @@ kubeletConfiguration:
     kubeAPIBurst: 10
     kubeAPIQPS: 5
     makeIPTablesUtilChains: true
+    managerMode: Watching
```

Comment:
Since "watching" is the default, why do you need to set it explicitly everywhere?

Reply:
Those are some kubeadm test data - it seems we are explicitly listing the whole config in mutliple places.
I didn't dig deeper into those kubeadm files, but the problem seems orthogonal to this PR.